src/features/Main/index.jsx

```diff
@@ -36,6 +37,7 @@ const Main = () => {
   const routes = [
     { path: '/dashboard', component: DashboardPage, exact: true },
     { path: '/students', component: StudentsPage, exact: true },
+    { path: '/classes/:className', component: ClassDetailPage, exact: true },
```

Comment:
Just a question here, why we use a `className` instead of `classId` what if someone changes the class name, back can find the selected class even when it original name changes? Just a question though

Reply:
The problem is the services  are filtered by class or course name, not by the id, it is something we have to change to avoid the recent bug for the encoded but that task was not prioritized, in the other story  I'm trying to fix this because we need to call the classes service anyway and with the class id could be work but it's scope of other story